.github/workflows/job_pytorch_models_tests.yml

```diff
@@ -156,6 +156,16 @@ jobs:
         env:
           TEST_DEVICE: CPU
 
+      - name: MoE Fusion Test
+        if: ${{ inputs.model_scope == 'precommit' }}
+        run: |
+          export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
+          python3 -m pip install optimum-intel==1.25.2 transformers==4.53.3 --upgrade
```

Comment:
That is a bad practice to introduce test dependencies in the  pipeline, why can't we use the common test requirements?

Reply:
Upgrade to transformers and optimum-intel is required to use versions that support qwen3_moe models.
Ideally it would be best to upgrade `tests/requirements_pytorch` to align versions, however this might be bit risky without thorough testing since this requirements are used in large amount of model tests, including nightly large models.
I've initially tried running other precommit suites `test_pa_transformation` `test_transformations` and it seem they were successful with upgraded versions